### DISCUSSION TITLE
OOM导致BE挂掉了

### DISCUSSION THREAD

**Discussion by 731879846kk** [apache/doris#13055] [A - General / Q&A]
本地虚拟机开了6G内存，创建了一个聚合模型表，导入CSV文件，大小在1.2G左右，导入后也成功了，但是我没有注意到是什么时候我的be 挂掉
![6f8f4a9f4d76fa724f4b0ad8e1d2bf6](https://user-images.githubusercontent.com/53074519/192967431-d98a70cb-58f7-4ab9-a046-6d8b76b6155c.jpg)
我想知道导入这样的csv文件需要设置什么参数来避免be被系统OOM掉？  还是说对csv文件的大小有要求？   之前看官网说最好不要超过2G， 我这个1.2G貌似也不是太大

**Comment by 731879846kk** [comment]
第二次尝试导入，导入完成显示load信息为success，然后 dmesg -T   又出现了 OOM。  难道是be需要的内存比较多吗，最少16G？
![image](https://user-images.githubusercontent.com/53074519/192971412-0864d76a-4a91-4edc-b2ed-94f9b3ed2f5d.png)

**Comment by dataalive** [comment]
比较奇怪的是，你给了6g内存，然后默认be最大是*0.8=4.8G内存，为啥杀的时候BE才1.8G左右。是不是FE混布着，FE多少内存啊。

这个是oom killer干掉了，因为发现整体内存不足，就杀socre分最高的

**Reply by 731879846kk** [reply]
我弄了3台虚拟机，每台都部署了Fe和Be，  内存的设置我没有修改过，就简单把 IP 和 数据目录设置了一下就创建表开始导入。      我这个是最新的版本 1.1.2的，之前1.1.1版本导入的时候还没发现这个问题。 当时导入文件导出什么的都正常，be也不会挂。现在这个我只要导入一次，这台机器的be就会被 kill 掉 ，fe 还正常，能正常登录去查看各个节点信息什么的
